vllm/distributed/device_communicators/shm_broadcast.py

```diff
@@ -453,23 +454,23 @@ def acquire_read(self,
                     self._read_spin_timer.spin()
 
                     # if we wait for a long time, log a message
-                    if (time.monotonic() - start_time
-                            > VLLM_RINGBUFFER_WARNING_INTERVAL * n_warning):
+                    elapsed = time.monotonic() - start_time
+                    if not indefinite and (elapsed
+                                           > VLLM_RINGBUFFER_WARNING_INTERVAL *
+                                           n_warning):
                         logger.info(
-                            ("No available shared memory broadcast block found"
-                             " in %s seconds. This typically happens when some"
-                             " processes are hanging, doing some time-consuming"
-                             " work (e.g. compilation), or sitting idle."),
-                            VLLM_RINGBUFFER_WARNING_INTERVAL,
-                        )
+                            "No available shared memory broadcast block found"
+                            " in %s seconds. This typically happens when some"
+                            " processes are hanging, doing some time-consuming"
+                            " work (e.g. compilation), or sitting idle.",
```

Comment:
I think it can't happen when idle now?

```suggestion
                            " processes are hanging or doing some time-consuming"
                            " work (e.g. compilation)",
```